exp_hub.html

```diff
@@ -58,7 +58,8 @@
 
 		// start the experiment
 		jsPsych.init({
-		  experiment_structure: experiments,
+		  //experiment_structure: experiments,
+		  timeline: experiments,
```

Comment:
ok I'll need to update [here as well](https://github.com/expfactory/expfactory-docker/blob/master/expdj/apps/turk/templates/mturk_battery.html#L72)